core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -72,6 +72,10 @@
     INSTANCE_NAME("instance.name", null, PropertyType.STRING, "Name of Accumulo instance to connect to"),
     INSTANCE_ID("instance.id", null, PropertyType.STRING, "UUID of Accumulo instance to connect to"),
 
+    // User information
```

Comment:
Seems like all of these props should go elsewhere if the goal is to deprecate ClientConfiguration in favor of Connector.builder()

Reply:
I moved all properties to new `ClientProperty` enum.